components/base/kinematicbase/fake_kinematics.go

```diff
@@ -184,7 +184,7 @@ func WrapWithFakePTGKinematics(
 		Base:            b,
 		frame:           frame,
 		parentFrame:     position.Parent(),
-		currentPosition: position.Pose(),
+		currentPosition: spatialmath.NewZeroPose(),
```

Comment:
TODO: once RSDK-5271 is merged this **must** be changed back!

Reply:
Once that PR is merged, this will cause lots of test failures, so I wouldn't worry about forgetting

Reply:
😆 🤣 true that!